README.md

```diff
@@ -1,11 +1,21 @@
 # commercetools Postman collections
 
-This repository contains Postman collections for the commercetools platform.
+This repository contains Postman collections for the commercetools APIs.
+
+## Why use postman collections?
```

Comment:
```suggestion
## Why use Postman collections?
```

Reply:
cool thanks !!👌😁, we are having fun, hope things are good with you too. ✌️